src/main/java/it/gov/pagopa/admissibility/utils/OnboardingConstants.java

```diff
@@ -28,4 +28,8 @@ private OnboardingConstants(){}
     public static final String CRITERIA_CODE_BIRTHDATE = "BIRTHDATE";
     public static final String CRITERIA_CODE_RESIDENCE = "RESIDENCE";
     //endregion
+
+    //region global sequence id
+    public static final String GLOBAL_SEQUENCE_ID = "GLOBAL_SEQUENCE_ID";
```

Comment:
let's use a semantic id representing the flow for which we are a creating a sequence
```suggestion
    public static final String GLOBAL_SEQUENCE_ID = "ANPR_E002_INVOKE";
```